src/Fixer/CastNotation/NoUnsetCastFixer.php

```diff
@@ -70,6 +70,11 @@ private function fixUnsetCast(Tokens $tokens, $index)
             return;
         }
 
+        $afterVar = $tokens->getNextMeaningfulToken($varIndex);
+        if (null === $varIndex || !$tokens[$afterVar]->equalsAny([';', [T_CLOSE_TAG]])) {
```

Comment:
Should be `null === $afterVar` probably?

Reply:
oh yes, copy-paste eror :man_facepalming: